test/test_torch.py

```diff
@@ -514,7 +514,8 @@ def test_mean_dim(self):
             self._test_dim_ops(
                 lambda t, d: t.mean(d),
                 lambda n, d: n.mean(d),
-                use_integral=False)
+                use_integral=False,
+                use_complex=True)
```

Comment:
This test doesn't run on CUDA. can you please extend the test for mean in `tensor_op_tests` to also test complex dtypes?

Reply:
Thanks for pointing this out, the tests for complex dtypes are added to `tensor_op_tests`.